<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>501</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="Calculator">
     <attribute name="title">
      <string>Calculator</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>471</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,1">
       <item>
        <widget class="QLineEdit" name="Input"/>
       </item>
       <item>
        <widget class="QPushButton" name="Read">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGraphicsView" name="Graphic">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>471</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Reset">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>390</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>160</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="PrecisionLayout">
       <item>
        <widget class="QSlider" name="Precision">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Precision_label">
         <property name="text">
          <string>Precision</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="minX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="maxX">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>340</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>361</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5">
       <item>
        <widget class="QLabel" name="Answer_label">
         <property name="text">
          <string>result :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Answer">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="Refresh">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>390</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>↻</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="minY">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>330</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="maxY">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Bank">
     <attribute name="title">
      <string>Bank</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>471</width>
        <height>421</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Credit</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>421</width>
          <height>261</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,10,0,0,0" columnstretch="1,1,1" rowminimumheight="0,0,0,0,0,0,0,0" columnminimumwidth="1,1,1">
         <item row="2" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>% за раз</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Процентная ставка</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Тип</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Переплата по кредиту</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Платеж в i-ый раз</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Сумма вклада</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>раз</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>единиц</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Срок</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>аннуитетный</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>дифференцированный</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Расчитать</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Общая выплата</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_4">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineEdit_5">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="lineEdit_6">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>единиц</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>единиц</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Deposit</string>
       </attribute>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>391</width>
          <height>371</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>340</y>
            <width>80</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Again</string>
          </property>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>20</y>
            <width>301</width>
            <height>101</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="2">
            <widget class="QLabel" name="label_33">
             <property name="text">
              <string>раз</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>Платеж</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_16"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_15"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>Произведенно в </string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_31">
             <property name="text">
              <string>единиц</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="ADD">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>120</y>
            <width>80</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>ADD</string>
          </property>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>190</y>
            <width>361</width>
            <height>141</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_4" rowstretch="1,1,1" columnstretch="1,1,1" rowminimumheight="1,1,1" columnminimumwidth="1,1,1">
           <item row="1" column="0">
            <widget class="QLabel" name="label_36">
             <property name="text">
              <string>Сумма налога</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_37">
             <property name="text">
              <string>единиц</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_35">
             <property name="text">
              <string>единиц</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_34">
             <property name="text">
              <string>Начисленные проценты</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_18">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_17">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_38">
             <property name="text">
              <string>Итоговая сумма на вкладе</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_39">
             <property name="text">
              <string>единиц</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_19">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QWidget" name="gridLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>381</width>
            <height>301</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1,1,1,1,10,0,0,0" columnstretch="2,0,0" rowminimumheight="1,1,1,1,1,10,0,0,0">
           <item row="7" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Налогом облагается часть &lt;br/&gt;дохода, вышедшая за&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="8" column="2">
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>единиц</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Капитализация процентов</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="lineEdit_14"/>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Налоговая ставка</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="3">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>раз</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEdit_12"/>
           </item>
           <item row="4" column="1">
            <widget class="QCheckBox" name="checkBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>раз</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Cумма вклада</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_10"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Налог каждые</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>% за</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_11"/>
             </item>
             <item>
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>раз</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Срок размещения</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Выплачивание каждые</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_7"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Процентная ставка</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_8"/>
           </item>
           <item row="7" column="2">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;единиц&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>раз</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="lineEdit_13"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_9"/>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>340</y>
            <width>80</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Log">
     <attribute name="title">
      <string>Log</string>
     </attribute>
     <widget class="QTextEdit" name="Loging">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>461</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Info">
     <attribute name="title">
      <string>Info</string>
     </attribute>
     <widget class="QLabel" name="About_Calculator">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>-20</y>
        <width>441</width>
        <height>451</height>
       </rect>
      </property>
      <property name="text">
       <string>sdsc\n\</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
